JobSeek/JobSeek.Api/Services/Implementation/BaseService.cs

```diff
@@ -15,49 +16,45 @@ public BaseService(IBaseRepository<TEntity> repository)
         {
             _repository = repository;
         }
-        public List<TEntity> GetAll()
-        {
-            return _repository.GetAll();
-        }
 
-        public List<TCustomEntity> GetAll<TCustomEntity>()
+        public virtual ListResponse<TCustomEntity> GetAll<TCustomEntity>()
```

Comment:
change the TCustomEntity to TEntity, and change GetAll<TCustomEntity>() to GetAll()